docs/formats/zng.md

```diff
@@ -627,19 +592,19 @@ where `<nelem>` and each symbol name is encoded as in an enum typedef.
 A map type value has the form:
 ```
 ----------------------------
-|0x23|<key-type>|<val-type>|
+|0x24|<key-type>|<val-type>|
 ----------------------------
 ```
 where `<key-type>` and `<val-type>` are recursive encodings of type values.
 
-#### 4.7 Alias Type Value
+#### 4.7 Type Name Type Value
```

Comment:
In `zed-data-model` you refer to this as a "Named Type". I would pick one for consistency sake.